src/index.html

```diff
@@ -7,12 +7,66 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="#"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <h1>Product cards</h1>  
+    <div
+      class="product__card"
+      data-qa="card"
+    >
+      <div class="product__image"></div>
+
+      <div class="product__info">
+        <div class="product__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="product__code">Product code: 195434</div>
+
+        <div class="product__review-box">
+          <div class="product__stars">
+            <div class="product__star product__star--highlighted"></div>
+            <div class="product__star product__star--highlighted"></div>
+            <div class="product__star product__star--highlighted"></div>
+            <div class="product__star product__star--highlighted"></div>
+            <div class="product__star"></div>
```

Comment:
The stars should be implemented using the `stars` block and `stars__star` element, not `product__stars` and `product__star`. Also, the modifier should be `stars__star--active` or similar, instead of `product__star--highlighted`. Adjust the class names according to the BEM and project requirements.